### ISSUE TITLE
SqueezeNet training accuracy stuck at 0.37 for face recognition task

### ISSUE THREAD

**Issue by madhurkashyap** [rcmalli/keras-squeezenet#16]
### Tensorflow Version

1.6

### Keras Version

2.1

### Keras-squeezenet Version

Latest

### Bug reports:

I am training SqueezeNet on my own face database created from YouTube videos. Apparently the training accuracy is not improving to this magic number of ~0.37. In order to deal with label noise I have tried to include 

Are there any known limitations or any help that can be provided would be useful.

### Code Sample:

sqnet = SqueezeNet();
sqnet.layers[-1].activation = relu
sqnet.layers[-1].name = 'fc_1'
out = Dense(6, activation='softmax',
            name='classifier')(sqnet.layers[-1].output)
sqnet = Model(sqnet.input, out)
for i in range(3,len(sqnet.layers)):
    sqnet.layers[-i].trainable = False
sqnet.compile(Adam(),loss='categorical_crossentropy',
                metrics=['acc'])
history = sqnet.fit_generator(
                      trg227,
                      steps_per_epoch=None,
                      epochs=2, verbose=1,
                      validation_data=valg227,
                      validation_steps=None,
                      workers=1,
                      use_multiprocessing=False,
                      shuffle=True,
                      initial_epoch=0)
